libcontainer/cgroups/fscommon/fscommon.go

```diff
@@ -39,6 +40,14 @@ func ReadFile(dir, file string) (string, error) {
 	return buf.String(), err
 }
 
+// FileExists returns true if a file exists in dir, otherwise false.
+func FileExists(dir, file string) bool {
```

Comment:
We have used `cgroups.PathExists` for similar cases before, see e.g. https://github.com/opencontainers/runc/blob/dcdf6b631da266153abba1e48493ffc218cf554b/libcontainer/cgroups/fs/memory.go#L74-L75

Reply:
Thanks! Switched to use `cgroups.PathExists` and removed `fscommon.FileExists`. Did not touch PathExists contents, yet @AkihiroSuda's [previous comment](https://github.com/opencontainers/runc/pull/3010#discussion_r648216865) seemed perfectly valid.